webpack.config.js

```diff
@@ -47,4 +47,7 @@ module.exports = {
       favicon: './public/favicon.ico',
     }),
   ],
+  node: {
+    fs: "empty"
```

Comment:
@kilometerskang What does this change do?

Reply:
I actually remember why I added that. It doesn't seem to break when I remove it so I'll just take it off